unpackaged/config/dev/profiles/Admin.profile

```diff
@@ -36,6 +36,9 @@
         <layout>Account-%%%NAMESPACE%%%HEDA Organization Layout</layout>
         <recordType>Account.%%%NAMESPACED_ORG%%%University_Department</recordType>
     </layoutAssignments>
+    <layoutAssignments>
+        <layout>%%%NAMESPACE%%%Application__c-%%%NAMESPACE%%%EDA Application Layout</layout>
```

Comment:
@lifewithryan @xinnanlisf It looks like we only maintain layout assignments in the admin.profile here for objects where we have more than one layout (for example, all objects where there are record types). Since we only have one layout for the Application object, I'm going to go ahead and remove these three lines.